dex/encrypt/encrypt.go

```diff
@@ -68,12 +68,12 @@ type argonParams struct {
 }
 
 // NewCrypter derives an encryption key from a password string.
-func NewCrypter(pw string) Crypter {
+func NewCrypter(pw []byte) Crypter {
 	return newArgonPolyCrypter(pw)
 }
 
 // Deserialize deserializes the Crypter for the password.
-func Deserialize(pw string, encCrypter []byte) (Crypter, error) {
+func Deserialize(pw []byte, encCrypter []byte) (Crypter, error) {
```

Comment:
Both arguments are of the same type now. You can remove the first `[]byte`.

`Deserialize(pw, encCrypter []byte)`